docs/getting-started/connecting-to-wordpress.md

```diff
@@ -0,0 +1,3 @@
+# Connect Frontity to Your WordPress Site
```

Comment:
Please, let's not create "coming soon" content

Reply:
I'm already working on it. I just wanted to separate the Welcome pull request from the Getting Started pull request - which is "coming soon". 😀

Reply:
then remove this file in this PR and add it in the Getting Started pull request

Reply:
Can we move on with this?

Reply:
I'm currently working on this file